dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapAdapter.java

```diff
@@ -207,6 +211,36 @@
      */
     private final AtomicReference<X509CertificateMapper> iCertificateMapperRef = new AtomicReference<X509CertificateMapper>();
 
+    /**
+     * Fetch the security service so we can load the KerberosService with any Kerberos config (keytab/config) for use if the bindAuthMechanism is GSSAPI/Kerberos
+     */
+    private final AtomicServiceReference<SecurityService> securityServiceRef = new AtomicServiceReference<SecurityService>("securityService");
```

Comment:
We don't use the SecurityService, so why are we relying on it? Seems we only use the KerberosService.